src/Regression/Client/Chrome.php

```diff
@@ -37,40 +43,59 @@ public function __construct(BrowserFactory $browserFactory, array $options, stri
      */
     public function send(RequestInterface $request, array $options = []): ResponseInterface
     {
-        //@todo add cache based on $options
-        /**
-         * use \HeadlessChromium\Exception\BrowserConnectionFailed;
-         *
-         * // path to the file to store websocket's uri
-         * $socket = \file_get_contents('/tmp/chrome-php-demo-socket');
-         *
-         * try {
-         * $browser = BrowserFactory::connectToBrowser($socket);
-         * } catch (BrowserConnectionFailed $e) {
-         * // The browser was probably closed, start it again
-         * $factory = new BrowserFactory();
-         * $browser = $factory->createBrowser([
-         * 'keepAlive' => true,
-         * ]);
-         *
-         * // save the uri to be able to connect again to browser
-         * \file_put_contents($socketFile, $browser->getSocketUri(), LOCK_EX);
-         * }
-         */
-        $browser = $this->browserFactory->createBrowser(
-            array_replace_recursive($this->options, $options)
-        );
-        $page = $browser->createPage();
+        $chromeClientOptions = new ChromeClientOptions();
+
+        if (!empty($options['chromeClientOptions']) && is_array($options['chromeClientOptions'])) {
```

Comment:
This will be redundant if we pass DTO instead of array in `$options['chromeClientOptions']`